src/library_trace.js

```diff
@@ -119,7 +119,7 @@ var LibraryTracing = {
     googleWTFEnterScope: function(name) {
       var scopeEvent = EmscriptenTrace.googleWTFData['cachedScopes'][name];
       if (!scopeEvent) {
-        scopeEvent = window.wtf.trace.events.createScope(name);
+        scopeEvent = window['wtf'].trace.events.createScope(name);
```

Comment:
FWIW I trying shimming this into our project (currently using version 1.38.47) and closure was having problems with variable name mangling:

* Usage of `window['wtf'].trace.events.createScope` was mangled
* Usage of `EmscriptenTrace.googleWTFData['cachedScopes']` wasn't mangled, but the declaration of `cachedScopes` was mangled.

Reply:
This probably needs something like `window['wtf']['trace']['events']`, or a Closure externs file. I have not used Google's WTF with closure (probably nobody else has either since this has not come up), probably needs fixing in several places.